### PR TITLE
Try to install all dependencies

### PR DESCRIPTION
**Pull request by elfosardo**
Remove if-else checks for commands presence and delegate dnf to install dependencies if needed.

### CODE REVIEW

File: bmctest.sh [hunk 1]
```diff
@@ -64,25 +64,9 @@ if ! sudo true; then
...
-        sudo dnf install python3-pip
-    fi
-    pip3 install yq
-fi
-if ! command -v parallel > /dev/null 2>&1; then
-    sudo dnf install -y parallel
-    echo "will cite" | parallel --citation > /dev/null 2>&1
-fi
+sudo dnf install -y curl nc podman python3-pip parallel
+python3 -m pip install yq
```
Comment (dtantsur) [inline bmctest.sh:68]: Hmm, how does this work at all without `sudo`?

Comment (elfosardo) [reply bmctest.sh:68]: the pip installation by default will do a per-user installation, if the system site-packages location is not writable

Comment (dtantsur) [reply bmctest.sh:68]: Oh, this is news to me. I guess it was changed at some point.

**Review by dtantsur — APPROVED** [review]

**Pull request merged**